tests/integration/lambdas/golang/handler.go

```diff
@@ -0,0 +1,20 @@
+package main
```

Comment:
I'd prefer if we can avoid checking in the large binary file (~8MB) into the repo. I think we should either compile the binary at runtime (if possible), or download it from elsewhere during the tests. (e.g., from our localstack-artifacts repo)

Reply:
Mmmm I think I'll download it during the tests, since compiling in will require having golang in the docker image or host.